core/utils/utils.go

```diff
@@ -515,3 +516,11 @@ func Uint256ToHex(n *big.Int) (string, error) {
 	}
 	return common.BigToHash(n).Hex(), nil
 }
+
+var dec10 = decimal.NewFromInt(10)
+
+func DecimalFromBigInt(i *big.Int, precision int32) decimal.Decimal {
+	raw := decimal.NewFromBigInt(i, 0)
```

Comment:
are we gonna change this?

Reply:
"are we gonna change this?" how come?  what's the context?

Reply:
Could we not use decimal.NewFromBigInt(..) instead? https://godoc.org/github.com/shopspring/decimal#NewFromBigInt